conf/core/ConfigOptions.asciidoc

```diff
@@ -1855,6 +1880,58 @@ This option allows for passing in a job ID for the currently executing command.
 trying to track multiple commands across a single job. When the Hootenanny API database is used
 for job output storage, the ID will be stored in the job status table of the associated map.
 
+=== josm.map.cleaner.add.detail.tags
+
+* Data Type: bool
+* Default Value: `false`
+
+If true, cleaning/validation detail tags will be added to map outputs when cleaning maps with JOSM.
+
+=== josm.map.validator.java.implementation
+
+* Data Type: string
+* Default Value: `hoot/core/josm/JosmMapValidator`
+
+File path relative to Hootenanny root directory without file extension or prefixed namespace to
+Java class to use as the JOSM Java map validator/cleaner implementation.
+
+=== josm.max.elements.for.map.string
+
```

Comment:
Maps are passed around by default as XML. I chose to go that route vs trying to create the JOSM map objects via JNI by memory due to how much more complex that was going to be. There's no clean entry point I could find into JOSM to create a map object and it does a lot of things behind the black box that I couldn't figure out how to replicate. However, passing around XML will obviously blow out the memory at some point, so this threshold decides whether we use file I/O to pass around the maps instead.